index.html

```diff
@@ -1312,6 +1323,7 @@ <h1>Speakers</h1>
 										<div class="social-links vertical-align">
 											<a target="_self" href="https://twitter.com/akshay_ratan"><i class="icon social_twitter"></i></a>
 											<a href="http://in.linkedin.com/pub/akshay-ratan/60/aa0/396" target="_self"><i class="icon social_linkedin"></i></a>
+											<a href="https://www.facebook.com/akshay.ratan" target="_self"><i class="icon social_facebook"></i></a>
```

Comment:
Keep the flow, the same.
Follow existing link structure.

Reply:
You can add target attribute before href. Even though, it much matters much, but adds to code quality!